frontend/src/js/controllers/authCtrl.js

```diff
@@ -142,8 +143,16 @@
                     onSuccess: function(response) {
                         if (response.status == 200) {
                             utilities.storeData('userKey', response.data.token);
+                            vm.redirectUrl = utilities.getData('redirectUrl');
```

Comment:
Got it, but please don't use local storage here. As I said earlier, try to implement this with Angular save state.